src/main/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePass.scala

```diff
@@ -118,7 +119,13 @@ class PropertiesFilePass(cpg: Cpg, projectRoot: String) extends ForkJoinParallel
   private def annotatedMembers() = cpg.annotation
     .fullName(".*Value.*")
     .where(_.member)
-    .map { x => (x.parameterAssign.head, x.member.head) }
+    .map { x =>
+      {
+        val param = if (x.parameterAssign.l.length > 0) { x.parameterAssign.head }
+        else { null }
```

Comment:
@rahul-privado instead of introduction `null`, you can try converting the `map` to  a `flatMap` and
```
if (x.parameterAssign.l.length >0)
         Some(param, x.member.head)
else 
         None

Reply:
Used a filter and just checked in. Please check

Reply:
Also earlier as I said `cpg.annotation.fullName(".*Value.*")` is not the correct way, it can match any annotation having `.*Value.*` in it, instead of Springframework's Value annotation.

Reply:
I have tried `.fullName("org.springframework.*Value")` but that does not work. Can you point me to the slack link where you have raised it with the joern team? I will follow up.